src/FortranChecks/f90sub/CMakeLists.txt

```diff
@@ -29,6 +29,12 @@ endif()
 # ---------------------------------------------------------------------------- #
 file( GLOB f90sources *.f90 *.F90 )
 
+# gfortran-10 needs to be told to allow 'invalid' initialization value.
+if( CMAKE_Fortran_COMPILER_ID STREQUAL GNU AND CMAKE_Fortran_COMPILER_VERSION
+  VERSION_GREATER_EQUAL "10.0" )
+  set_source_files_properties( drel.f90 PROPERTIES COMPILE_FLAGS "-fallow-invalid-boz" )
+endif()
```

Comment:
This test and the `Draco_MPI` test below initialize data with hex data as a stress test.  gfortran-10 warns about the 'invalid' data assignment, but we actually want it. This suppresses the warning.